lighthouse-cli/run.js

```diff
@@ -195,6 +195,25 @@ async function potentiallyKillChrome(launchedChrome) {
   });
 }
 
+/**
+ * @param {string} url
+ * @param {LH.CliFlags} flags
+ * @param {LH.Config.Json|undefined} config
+ * @param {ChromeLauncher.LaunchedChrome} launchedChrome
+ * @return {Promise<LH.RunnerResult|undefined>}
+ */
+async function runLighthouseWithFraggleRock(url, flags, config, launchedChrome) {
+  const fraggleRock = require('../lighthouse-core/fraggle-rock/api.js');
+  const puppeteer = require('puppeteer');
+  // @ts-expect-error - FIXME: lighthouse-logger is greedy and takes over `debug` for all packages
+  require('debug').enable('-puppeteer:*');
```

Comment:
Please pull up this require to reduce an extra complication when automating the conversion to ESM.

Reply:
Oh is that happening imminently? I don't want this line to exist for more than a week (PR already in progress to make it unnecessary)

If you're referring to puppeteer, I was hoping to avoid injecting puppeteer into the default dependencies for an experimental flag due to the extra postinstall hooks. `const puppeteer = await import('puppeteer')` should be OK, right?

Reply:
> Oh is that happening imminently?

No, just a mechanical code review suggestion I have these days when I see `require` :)

> const puppeteer = await import('puppeteer') should be OK, right?

Yeah